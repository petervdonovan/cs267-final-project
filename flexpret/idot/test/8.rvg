[mu []
[def (Areg) t0]
[def (A) [array-new Areg of length 32]]
[[file {Reset_Handler} {
  Reset_Handler:
    [id init]
    [single-core t5
      [pariterate-hart t1 of [rng-seeded t2 t3 bits 5 length 32 seeded 7]
        {
          [critical-hart t0 t5
            {
              [sim-print t1 t4]
            }]
        }]]
    [id die]}]]
]
