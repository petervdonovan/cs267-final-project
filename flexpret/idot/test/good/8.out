Core-0:         26
Core-0:         26
Core-0:          0
Core-0:         20
Core-0:         14
Core-0:         14
Core-0:         20
Core-0:         11
Core-0:          9
Core-0:          3
Core-0:         10
Core-0:          0
Core-0:          9
Core-0:          3
Core-0:         10
Core-0:         29
Core-0:          8
Core-0:         30
Core-0:         22
Core-0:          8
Core-0:          0
Core-0:         22
Core-0:         30
Core-0:          1
Core-0:         20
Core-0:          0
Core-0:         20
Core-0:         14
Core-0:         26
Core-0:         14
Core-0:         26
Core-0:          2
Core-0 is done
Core-1 is done
Core-2 is done
Core-3 is done
Core-4 is done
Core-5 is done
Core-6 is done
Core-7 is done
Core-8 is done
Core-9 is done
Core-10 is done
Core-11 is done
Core-12 is done
Core-13 is done
Core-14 is done
Core-15 is done
All cores are done terminating
Assertion failed: Program terminated sucessfully
